content/en/blog/2023-02-01-january-update/index.md

```diff
@@ -0,0 +1,415 @@
+---
+author: dholbach
+date: 2023-01-24 09:30:00+00:00
+title: January 2023 Update
+description: "XXX"
+url: /blog/2023/02/january-2023-update/
+tags: [monthly-update]
+resources:
+- src: "**.{png,jpg}"
+  title: "Image #:counter"
+---
+
+As the Flux family of projects and its communities are growing, we
+strive to inform you each January about what has already landed, new
+possibilities which are available for integration, and where you can get
+involved. Read our last update [here](/blog/2022/12/november-2022-update/).
+
+It's the beginning of February 2023 - let's recap together what
+happened in December and January - it has been a lot!
+
+## News in the Flux family
+
+### Next Flux release: more stability and performance improvements
+
+We have released Flux v0.38. Users are encouraged to upgrade for the best
+experience.
+
+Features and improvements
+
+- Graduation of Notification APIs to `v1beta2`, to upgrade please see
+  [the release notes](https://github.com/fluxcd/flux2/releases/tag/v0.38.0).
+- Support for defining Kustomize components with `Kustomization.spec.components`.
+- Support for piping multi-doc YAMLs when publishing OCI artifacts with
+  `kustomize build . | flux push artifact --path=-`.
+- Support for Gitea commit status updates with `Provider.spec.type` set to
+  `gitea`.
+- Improve the memory usage of `helm-controller` by disabling the caching of
+  `Secret` and `ConfigMap` resources.
+- Update the Helm SDK to v3.10.3 (fix for Helm CVEs).
+- All code references to `libgit2` were removed, and the
+  `GitRepository.spec.gitImplementation` field is no longer being honored.
+
+Documentation
+
+The official example repository was refactored. The new version comes with
+the following improvements:
+
+- Make the example compatible with ARM64 Kubernetes clusters.
+- Add Weave GitOps Helm release to showcase the Flux UI.
+- Replace the ingress-nginx Bitnami chart with the official one that contains
+  multi-arch container images.
+- Add cert-manager Helm release to showcase how to install CRDs and custom
+  resources using `dependsOn`.
+- Add Let’s Encrypt ClusterIssuer to showcase how to patch resources in
+  production with Flux `Kustomization`.
+- Add the `flux-system` overlay to showcase how to configure Flux at
+  bootstrap time.
+
+♥ Big thanks to all the Flux contributors that helped us with this release!
+
```

Comment:
I would like to release the blog post tomorrow. Shall we give an outlook of the upcoming Flux release? c.f. https://github.com/fluxcd/flux2/issues/3533

cc @stefanprodan